openpype/modules/sync_server/sync_server_module.py

```diff
@@ -1616,6 +1612,10 @@ def reset_site_on_representation(self, collection, representation_id,
 
         elem = {"name": site_name}
 
+        query = {
```

Comment:
Side notes:
It looks like all the methods `_reset_site_for_file`, `_reset_site`, `_remove_site` and `_pause_unpause_site` could expected representation id instead of `query`. There is no other query filter then representation id.

And variables `collection` should be renamed to `project_name`.